build.gradle

```diff
@@ -11,8 +11,6 @@ plugins {
 group 'fi.aalto.cs'
 version '2.0'
 
-sourceCompatibility = 11
```

Comment:
What's the effect of this?

Reply:
seems that Gradle will use the version of Java that the OS defaults to
```bash
export JAVA_HOME=`/usr/libexec/java_home -v 11`
```

Reply:
otherwise, things build and start and work

Reply:
Wouldn't it be safer to explicitly specify version 11 instead of assuming that's the version that the OS has installed?

Reply:
for some reason, IDEA tells, that `sourceCompatibility = 11` assignment is not used, but true that, if set to `1.8` then it does not build, so rolling this back